warp/_src/builtins.py

```
@@ -10125,6 +10140,8 @@ def tile_cholesky_generic_lto_dispatch_func(
         num_threads = options["block_dim"]
         parameter_list = f"({dtype}*, int*)"
         req_smem_bytes = a.type.size * type_size_in_bytes(a.type.dtype)
```

Ruff (0.14.2)

`type_size_in_bytes` may be undefined, or defined from star imports

(F405)